### PR TITLE
Add 3.3 Release Notes

### PR DESCRIPTION
**Pull request by bbuckingham**
This PR will replace: https://github.com/theforeman/theforeman.org/pull/796

Stub for 3.3 release notes, taken from (https://goo.gl/AqRGv3). @jlsherrill mind doing a review of possible release note worth items as well? Will work on/seek out descriptions after we settle.

### CODE REVIEW

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 1]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
+
+### Publish "Latest" Version in Composite Content Views
+
+### Capsule Download Policy
+
+### Tracer Support
+
```
Comment (jlsherrill) [inline plugins/katello/3.3/release_notes/release_notes.md:44]: After updating some packages on a host sometimes a service will need to be restarted in order for the new version to take affect.  Tracer is a utility to help detect these services and Katello 3.3 supports hosts reporting this information.  Simply update to the newest katello-agent and these reports will be sent whenever a package is updated.  You can see what services need to be restarted and restart them via Remote Execution from the new Content Host Details Traces page.

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 2]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
+### Improved Manifest Import and Delete
+
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
+
+### Publish "Latest" Version in Composite Content Views
+
+### Capsule Download Policy
```
Comment (jlsherrill) [inline plugins/katello/3.3/release_notes/release_notes.md]: Actually this should be called "Smart Proxy Content".  In fact we should probably include this as a feature so its more clear. 

Smart Proxies with content can now be configured with their own download policy.  This allows the main server to sync the full contents of its configured repositories while the Smart Proxy can be configured to use an "On Demand" policy and only sync what is actually used by clients.

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 3]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
 
 ## Features
 
+### Updated UI
+
+### Improved Manifest Import and Delete
+
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
```
Comment (jlsherrill) [inline plugins/katello/3.3/release_notes/release_notes.md]: You might group this one and the next with "Content View Improvements" and add:

* Arch available for content view package filters
* Allow Composite content views to be publish containing components with different copies of the same repositories, combining the contents as part of the composite.

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 4]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
-# Katello Nightly () Release Notes
+# Katello 3.3 (Baltic Porter) Release Notes
 
-{% if page.version and page.version != '3.3' %}
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
 
 ## Features
```
Comment (jlsherrill) [inline plugins/katello/3.3/release_notes/release_notes.md:12]: New Feature:  Package Applicability

See what packages need updating on your host.  Previously only available via repositories with Errata, now you can see exactly what updates are available for each host and apply them directly.

File: plugins/katello/3.3/release_notes/release_notes.md (hunk 3 above)
Comment (daviddavis) [reply plugins/katello/3.3/release_notes/release_notes.md]: Promotion to multiple environments allows users to promote a content view to multiple environments at once. Before 3.3, users could only promote a content view version to one environment at a time. Right now this feature is only available via the API and CLI.

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 5]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
 
+### Updated UI
+
+### Improved Manifest Import and Delete
```
Comment (daviddavis) [inline plugins/katello/3.3/release_notes/release_notes.md:17]: Manifest import and delete actions were refactored for 3.3 in anticipation of Candlepin 2.0. Now users should be able to see a step-by-step breakdown of tasks for these actions in the Dynflow console.

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 6]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
+
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
+
+### Publish "Latest" Version in Composite Content Views
+
```
Comment (jlsherrill) [inline plugins/katello/3.3/release_notes/release_notes.md:34]: When configuring a Composite Content View a specific version of each component is selected.  Now you can simply select 'Latest' and at publish time the latest version will be calculated and used within the composite.

File: plugins/katello/3.3/release_notes/release_notes.md (hunk 4 above)
Comment (bbuckingham) [reply plugins/katello/3.3/release_notes/release_notes.md:12]: Added

File: plugins/katello/3.3/release_notes/release_notes.md (hunk 5 above)
Comment (bbuckingham) [reply plugins/katello/3.3/release_notes/release_notes.md:17]: Added

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 7]
```diff
@@ -4,16 +4,42 @@ title: Release Notes
...
 ## Features
 
+### Updated UI
+
+### Improved Manifest Import and Delete
+
+Manifest import and delete actions were refactored for Katello 3.3 in anticipation of Candlepin 2.0. Now users are able to see a step-by-step breakdown of tasks for these actions in the Dynflow console.
+
+### Username and Password for Repositories
+
```
Comment (parthaa) [inline plugins/katello/3.3/release_notes/release_notes.md:22]: Can set the username/password to access upstream yum/puppet repositories or docker registries that require http basic authentication.

Comment (parthaa) [reply plugins/katello/3.3/release_notes/release_notes.md:22]: Or may be

 Allows for the setting of username and password when syncing authenticated http and https repositories

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 8]
```diff
@@ -4,16 +4,26 @@ title: Release Notes
...
 
-{% if page.version and page.version != '3.3' %}
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
 
 ## Features
 
+### Updated UI
```
Comment (waldenraines) [inline plugins/katello/3.3/release_notes/release_notes.md:13]: Simplify UI by moving towards a single item view in accordance with patternfly best practices and in preparation for vertical for upcoming vertical navigation.

Comment (bbuckingham) [reply plugins/katello/3.3/release_notes/release_notes.md:13]: Added

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**